src/pages/Delivery/index.jsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
Are there more errors than the ones below?

Reply:
In this file, the errors are the same